ee/backend/app/src/main/kotlin/io/tolgee/ee/service/TaskService.kt

```diff
@@ -559,8 +559,8 @@ class TaskService(
   private fun createNotificationIfApplicable(task: Task) {
     val notificationType =
       when (task.state) {
-        TaskState.DONE -> NotificationType.TASK_COMPLETED
-        TaskState.CLOSED -> NotificationType.TASK_CLOSED
+        TaskState.FINISHED -> NotificationType.TASK_COMPLETED
```

Comment:
Let's migrate also the notification types. 

Also if you are in the migrating this, the `TASKS_CREATE` should be renamed to `TASK_CREATE`.

Reply:
There are to activity events, `TASK_CREATE` and `TASKS_CREATE`, because you can create multiple tasks at once, I think it's correct that it's separated, or not?

Reply:
OK then.